src/lib/divider/divider.spec.ts

```diff
@@ -0,0 +1,99 @@
+import {async, TestBed} from '@angular/core/testing';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+import {MatDividerModule} from './divider-module';
+
+
+describe('MatDivider', () => {
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MatDividerModule],
+      declarations: [
+        MatDividerBase,
+        MatDividerVertical,
+        MatDividerInset,
+        MatDividerVerticalInset,
+        MatDividerWithCssClass,
+      ],
+    });
+
+    TestBed.compileComponents();
+  }));
+
+  it('should not apply any additional class to a list without lines', () => {
+    const fixture = TestBed.createComponent(MatDividerBase);
```

Comment:
Instead of calling `TestBed.createComponent` in every test, set up a `beforeEach` block to do this setup.

let fixture: ComponentFixture<MatDividerTestComponent>;

beforeEach(() => {
  fixture = TestBed.createComponent(MatDividerTestComponent);
});

Reply:
Will add